x-pack/plugins/remote_clusters/public/sections/remote_cluster_form/remote_cluster_form.js

```diff
@@ -109,7 +109,14 @@ export class RemoteClusterFormUi extends Component {
       errors.name = (
         <FormattedMessage
           id="xpack.remoteClusters.form.errors.nameMissing"
-          defaultMessage="Name is required."
+          defaultMessage="Nickname is required."
```

Comment:
Yeah not easy the wording. What about "Cluster alias"? And then the error would say, "You need to provide an alias name for your cluster". To me, _alias_is very clear, maybe not for everyone though.

Reply:
Ah, actually this was leftover from when I was considering using "Nickname". I kept it as "Name" because I updated the description text to make it clear that users can choose whatever name they want. I'll revert this back. Good catch!

Reply:
Alias is nice too, but I'm concerned about people thinking they have a special meaning since "Index aliases" and "Field aliases" are already special terms in the context of Elasticsearch. I think we're OK with just "Name" for now.